webpages/profile/profile.html

```diff
@@ -27,11 +27,13 @@
   <h2>User Profile</h2>
   <img class="photo" src="/img/user.png" alt="photo of the person">
   <h2 class="only localediting">Editing Locally</h2>
+  <div class="username">Username: <span></span>&nbsp;<a class="changebutton" href="/register">change</a></div>
   <div class="name">loading&hellip;</div>
   <div class="email"></div>
   <div class="joined">Joined on <span class="date">&hellip;</span></div>
   <p class="only localediting">These are the things you have editing locally on your computer:</p>
   <p class="only localediting resetlocalstorage">(reset local storage)</p>
+  <p class="only localediting register">Register for online storage</p>
```

Comment:
we might have "register to add a comment" in place of "new comment" everywhere when not logged in